Google Scholar.js

```diff
@@ -887,11 +909,33 @@ ItemFactory.prototype.getAttachments = function (doctype) {
 	attachments = [];
 	for (i = 0, ilen = this.attachmentLinks.length; i < ilen; i += 1) {
 		if (!this.attachmentLinks[i]) continue;
-		attachments.push({
-			title:"Google Scholar Linked " + doctype,
-			url:this.attachmentLinks[i],
-			type:"text/html"
-		});
+		if (!Zotero.Utilities.setMultiField || "string" === typeof this.attachmentLinks[i]) {
```

Comment:
`!Zotero.Utilities.setMultiField` is sort of redundant here, since, in non-MLZ environment, `.attachmentLinks[i]` is always a string. It also makes it seem like you could be entering this block where `.attachmentLinks[i]` is _not_ a string and that would be bad.